src/dodal/devices/ophyd_async_xspress3_mini/xspress3_mini_channel_oa.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+from ophyd_async.core import Device
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class TimeSeriesValues(str, Enum):
+    START_VALUE = "Acquire"
+    STOP_VALUE = "Done"
+    UPDATE_VALUE = ""
+
+
+class Xspress3MiniChannel(Device):
+    def __init__(self, prefix: str, name: str = "") -> None:
+        # Define some signals
+        self.update_arrays = epics_signal_rw(float, prefix + "SCAS:TS:TSAcquire")
+
+        self.roi_high_limit = epics_signal_rw(float, prefix + "SCA5_HLM")
```

Comment:
What is roi?

Reply:
region of interest, you can think of xspress3 like a smart band pass filter so you have to tell it the lower and upper limit for it to work.